### ISSUE TITLE
[FEATURE] Option to hide default value when its type is `pydantic.BaseModel`

### ISSUE THREAD

**Issue by lenormf** [mansenfranzen/autodoc_pydantic#149]
Hi,

Whenever describing a model with `autopydantic_model`, I want default values to be shown.

However, some fields of the model point to another model (which inherits `pydantic.BaseModel`).

That results in that model’s representation (along with *all* its fields and subfields) being shown as a default value, spawning several lines.

I would like to somehow disable that behaviour, as I can already click on the hyperlinked type of that problematic field to learn about its own fields.

Hope it’s clear,

Thanks!

**Comment by mansenfranzen** [comment]
Hi @lenormf,

thanks for raising the question!

Have you looked at this [FAQ section](https://autodoc-pydantic.readthedocs.io/en/latest/users/faq.html#show-inherited-fields-validators)? It might very well answer your question.

**Comment by ecciv** [comment]
Hi @mansenfranzen, If I understand the issue correctly, then no solution is given in the FAQ.

If a fields default value is another pydantic dataclass, then the default value for the field can become excessively long, especially when there is multiple layers of dataclasses. If I've understood this issue correctly, @lenormf is having the same problem that I am in https://github.com/mansenfranzen/autodoc_pydantic/issues/322.

**Comment by lenormf** [comment]
Correct, we are having the same issue, albeit with different symptoms (I want to hide default values that point to another model).
